case-lib/apause.exp

```diff
@@ -137,8 +137,20 @@ set volume_always_zero true
 
 expect {
 
+    # `man re_syntax` claims that Tcl regular expressions are compliant
+    # with the basic and extended POSIX ones while adding a 3rd,
+    # extended flavor.  It's not clear which flavor `expect -re` uses
+    # but it's not the basic one.
+    # Use {} not "" to avoid quoting issues and backslash proliferation.
+
     # When multiple patterns match, first pattern wins.
 
+    -nocase -re {error.*\r|PAUSE.*no[[:blank:]]*hw[[:blank:]]*support.*\r} {
+        set buffer_with_lf "[cr_to_lf $expect_out(buffer)]"
+        log 0 "ERROR: $buffer_with_lf"
+        exit 1
+    }
+
```

Comment:
just to be clear on the commit message: PAUSE will be disabled unless we have a kernel parameter that keeps it enabled. All CI platforms SHALL keep the pause enabled.

Reply:
That's what I tried to say but more briefly and without getting into Intel CI specifics. Too briefly? I mean, should I change the commit message or is your comment here enough?

Reply:
thats' fine, just wanted to check we were aligned.